### DISCUSSION TITLE
Change name - Round 2

### DISCUSSION THREAD

**Discussion by daudix** [GradienceTeam/Gradience#127] [Polls]
Recently, a member of GNOME asked us to change the name because Adwaita Manager look like an official app.

So, we found some names, and after [Round 1](https://github.com/AdwCustomizerTeam/AdwCustomizer/discussions/113), here is a more filtred list

**[NOTE]** Team, project and [matrix chat](https://matrix.to/#/#AdwCustomizer:matrix.org) names will also be changed

## Thanks everyone for voting 👍 vote is ended, and a winner is Gradience!

**Comment by 0xMRTT** [comment]
Hard to choose between gradience and paint roller

**Reply by daudix** [reply]
We can publish a post on GNOME Reddit for more objective results

Edit: [Done!](https://www.reddit.com/r/gnome/comments/wnosd7/new_name_for_adwaita_manager/)

**Reply by tseli0s** [reply]
Didn't we already go with Gradience? I thought I saw it in some labels. Anyways, whatever the name is, leave it to me to change the source, so I get to commit something :P
